src/lib/y2firstboot/dialogs/wsl_product_selection.rb

```diff
@@ -87,17 +95,30 @@ def dialog_content
             # TRANSLATORS: check box label
             Left(CheckBox(Id(:wsl_gui_pattern),
               _("Install WSL GUI pattern (requires registration)"),
-              wsl_gui_pattern))
+              wsl_gui_pattern)),
+            # TRANSLATORS:
+            Label(_("The WSL systemd pattern provides wsl.conf adjustment and init symlink for\n" \
```

Comment:
The checkbox label is IMHO too technical, that should be part of the documentation and in the help text.

For label I'd prefer some simpler, less technical description understandable for more users.

Something like "The WSL systemd pattern allows using the Linux services inside a WSL container."

Reply:
it is kind of similar to the one about WSL GUI. I am fine with simplifying both, but I am not sure about who is target user.

Reply:
I agree with @lslezak. I would something like he proposed.

Reply:
well, goal of the PBI is to use similar wording like in pattern description and I also ask on discuss-wsl on slack for feedback. To be honest I am not sure about terms used in WSL like if we should talk about services or container in WSL context.

Reply:
Ok, well, we can change it if somebody complains.